docs/API_AND_INTEGRATION_GUIDE.md

```
@@ -203,6 +203,125 @@ GET /api/v1/kills/count/{system_id}
 ```
 
+### Server-Sent Events (SSE) Stream
+
+```http
+GET /api/v1/kills/stream
+```
+
+Real-time streaming of killmail data using Server-Sent Events protocol. Perfect for server-side applications that need a simple, unidirectional stream of killmail updates.
+
+**Query Parameters:**
+- `system_ids` - Comma-separated list of system IDs to filter (e.g., `30000142,30000144`)
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)